### ISSUE TITLE
C-k / C-j / C-g / C-S-g / C-q for moving around and exiting?

### ISSUE THREAD

**Issue by aldanor** [mgunyho/tere#35]
Currently, some home-row shortcuts have alt as a modifier, which may be much less convenient than ctrl (e.g. if you have caps lock to ctrl). Would it be possible to also support C-k, C-j, C-g, C-S-g, C-q and some other shortcuts with ctrl modifier as well?

**Comment by mgunyho** [comment]
Customizing keyboard shortcuts is currently not implemented, but it's certainly high on the todo list! For now, the best you can do is to change the `modifiers == ALT` to `modifiers == CONTROL` [here](https://github.com/mgunyho/tere/blob/a8d71bf7defc2daee824e44d9b314b524c9a2889/src/ui/mod.rs#L668-L679) and recompile. Sorry if that's too much of a hassle for you.

I will be offline for around two weeks so I won't have time to look into this very soon, but I'll see after that.

**Comment by zbindenren** [comment]
I was thinking exactly the same thing. This would be more vim like. I try to compile it.

**Comment by mgunyho** [comment]
Custom keyboard shortcuts have now been implemented in #57, which will be released soon. With this, you can add
```shell
--map ctrl-h:ChangeDirParent,ctrl-j:CursorDown,ctrl-k:CursorUp,ctrl-l:ChangeDir

# or if you prefer
--map ctrl-h:ChangeDirParent \
--map ctrl-j:CursorDown \
--map ctrl-k:CursorUp \
--map ctrl-l:ChangeDir
```
to your shell config to map ctrl+hjkl for navigation.

If you want to disable the `Alt-*` mappings, you can do
```
--map alt-h:None,alt-j:None,alt-k:None,alt-l:None
```
